### ISSUE TITLE
Deleting records should be synchronised

### ISSUE THREAD

**Issue by yatharthranjan** [RADAR-CNS/RADAR-RedcapIntegration#15]
Deleting records should be synchronised between redcap and management portal so we don't have to manually delete from both portals

**Comment by yatharthranjan** [comment]
On second thoughts, this might not be good idea, since in Redcap project admins have right to delete the records but in MP only the system admin can delete them. So better to do it manually and allow project admins to mark the subject discontinued. Even in Redcap, this should be marked withdrawn. We can instead add a flow to read the withdrawn field in redcap and discontinue the participant in MP

**Comment by mpgxvii** [comment]
Yeah I was wondering about this since we now update attributes, maybe in the future we can add the withdraw field as well to update the subject's status in MP.
